test/gtest/uct/test_peer_failure.cc

```diff
@@ -538,3 +540,68 @@ UCS_TEST_SKIP_COND_P(test_uct_peer_failure_multiple, test,
 }
 
 UCT_INSTANTIATE_TEST_CASE(test_uct_peer_failure_multiple)
+
+class test_uct_peer_failure_keepalive : public test_uct_peer_failure
+{
+protected:
+    void is_skipped()
+    {
+        if (!has_transport("rc_mlx5")) {
+            UCS_TEST_SKIP_R("Unsupported");
+        }
+    }
+};
+
+UCS_TEST_P(test_uct_peer_failure_keepalive, killed)
+{
+    ucs_status_t status;
+
+    is_skipped();
+
+    scoped_log_handler slh(wrap_errors_logger);
+    flush();
+    EXPECT_EQ(0, m_err_count); /* it works !!! */
```

Comment:
what is working?
comment a bit confused..

Reply:
comment is removed